psydac/api/postprocessing.py

```diff
@@ -139,19 +154,35 @@ class OutputManager:
 
     _current_hdf5_group :  h5py.Group
         Group where the fields will be saved in the next ``export_fields``.
-    
+
     _static_names : list
+        List of the names of the statically saved fields.
+
+    _space_types_to_str : dict
+
+    _spaces_names : list
+        List of the names of the saved spaces.
+
+    _comm : mpi4py.MPI.Intracomm or None
+
+    _mode : str in {'r', 'r+', 'w', 'w-', 'x', 'a'}
+
+    _fields_file : h5py.File
+
+    _spaces_types_to_str : dict
+        Dictionnary from Sympde space Datatypes
```

Comment:
Typo (7 x in this file): Dictionnary/ies -> Dictionary/ies

Reply:
Fixed in [7802f17](https://github.com/pyccel/psydac/pull/230/commits/7802f1727d614941a1cd1caf1810a483d405921c)